payjoin-cli/src/db/v2.rs

```diff
@@ -90,9 +90,22 @@ impl SessionPersister for SenderPersister {
     fn close(&self) -> std::result::Result<(), Self::InternalStorageError> {
         let conn = self.db.get_connection()?;
 
+        let completed_event_id: Option<i64> = conn
+            .query_row(
+                "SELECT id FROM send_session_events 
+             WHERE session_id = ?1 
+             ORDER BY id DESC LIMIT 1",
+                params![*self.session_id],
+                |row| row.get(0),
+            )
+            .ok();
+
+        let completed_event_id = completed_event_id
+            .ok_or_else(|| format!("completed_event_id for Session {} is none", self.session_id))?;
```

Comment:
Does it make sense to `panic!` here? This would indicate a bug in the API and/or CLI.

Also, I would just make a new error variant for this as oppose to converting a string to some random sqlite error variant